testpar/t_filters_parallel.c

```diff
@@ -1287,6 +1291,177 @@ test_write_filtered_dataset_interleaved_write(void)
     return;
 }
 
+/*
+ * Tests parallel write of transformed and filtered data
+ * in the case where only one process is writing to a
+ * particular chunk in the operation. Normally, a data
+ * transform function will cause the parallel library to
+ * break to independent I/O and this isn't allowed when
+ * there are filters in the pipeline. However, in this
+ * case the parallel library recognizes that the used
+ * data transform function "x" is the same as not applying
+ * the transform function. Therefore it does not apply
+ * the transform function resulting in not breaking to
+ * independent I/O.
+ *
```

Comment:
Please add the comment to the RELEASE.txt file (similar to what you have here) explaining the change. It should go under Bug Fixed since HDF5 ... / Library.
Thank you!

Reply:
@epourmal , I have added a comment about the implemented improvement to the `RELEASE.txt` file. Thanks for mentioning this.

Reply:
Thank you! We are currently testing on HPC systems now and will accept the change.